src/gui/state/game/game.cpp

```diff
@@ -720,7 +720,20 @@ std::vector<Message> Game::GetMessages() const {
 
   static constexpr size_t SAFE_MAX_ACTIVE_NORMAL_PLUGINS = 254;
 
-  if (activeNormalPluginsCount > SAFE_MAX_ACTIVE_NORMAL_PLUGINS &&
+  if (activeNormalPluginsCount > SAFE_MAX_ACTIVE_NORMAL_PLUGINS && settings_.Type() == GameType::tes3 && 
+      std::filesystem::exists(preludePath_/"MWSE.dll")) {
+      auto logger = getLogger();
+    if (logger) {
+      logger->warn(
+          "More than 255 normal plugins are activated at the same time.");
+    }
+    output.push_back(PlainTextMessage(
+        MessageType::warn,
+        boost::locale::translate(
+          "Do not launch Morrowind without the use of MWSE or it will"
+          "cause unaccountable damage to your game.")));
```

Comment:
I'm a bit confused by the use of `unaccountable` here, I don't know what it's trying to communicate (though I'm not a Morrowind mod user, so maybe that's why). Does it mean that it won't obvious why things break when they do?